stdlib/tkinter/__init__.pyi

```diff
@@ -1112,17 +1115,169 @@ class Canvas(Widget, XView, YView):
     def canvasx(self, screenx, gridspacing: Optional[Any] = ...): ...
     def canvasy(self, screeny, gridspacing: Optional[Any] = ...): ...
     def coords(self, *args): ...
-    def create_arc(self, *args, **kw): ...
-    def create_bitmap(self, *args, **kw): ...
-    def create_image(self, *args, **kw): ...
-    def create_line(self, *args, **kw): ...
-    def create_oval(self, *args, **kw): ...
-    def create_polygon(self, *args, **kw): ...
-    def create_rectangle(self, *args, **kw): ...
-    def create_text(self, *args, **kw): ...
-    def create_window(self, *args, **kw): ...
+    def create_arc(self, *args, **kw) -> _CanvasObjectId: ...
+    def create_bitmap(self, *args, **kw) -> _CanvasObjectId: ...
+    def create_image(self, *args, **kw) -> _CanvasObjectId: ...
+    def create_line(
+        self,
+        __x0: float,
+        __y0: float,
+        __x1: float,
+        __y1: float,
+        *,
+        activedash: _Color = ...,
+        activefill: _Color = ...,
+        activestipple: str = ...,
+        activewidth: Union[float, str] = ...,
+        arrow: Literal["first", "last", "both"] = ...,
+        arrowshape: tuple[float, float, float] = ...,
+        capstyle: Literal["round", "projecting", "butt"] = ...,
+        dash: Union[tuple[float], tuple[float, float], tuple[float, float, float, float]] = ...,
+        dashoffset: Union[float, str] = ...,
+        disableddash: _Color = ...,
+        disabledfill: _Color = ...,
+        disabledstipple: _Bitmap = ...,
+        disabledwidth: Union[float, str] = ...,
+        fill: _Color = ...,
+        joinstyle: Literal["round", "bevel", "miter"] = ...,
+        offset: Union[float, str] = ...,
+        smooth: Literal[0, 1] = ...,
```

Comment:
Usually `bool` is best for things like this. There's more discussion about this in #4669.

```suggestion
        smooth: bool = ...,
```